zerver/openapi/zulip.yaml

```diff
@@ -2636,6 +2636,11 @@ paths:
                       type: string
                       description: |
                         The version of Zulip running in the server.
+                    zulip_feature_level:
+                      type: string
+                      description: |
+                        An integer to indicate the features added to the server. It is
+                        finer-grained than the version number.
```

Comment:
Is the intent to have this be a number, not a string?  If so, we should make it one.

Reply:
I agree; the `type` should be `integer` here. Thanks!